internal/tcpchecker/portchecker.go

```diff
@@ -0,0 +1,32 @@
+package tcpchecker
+
+import (
+	"time"
+
+	"github.com/prometheus/common/log"
+)
+
+type PortChecker struct {
+	address string
+	port    int
+}
+
+func StartPortChecking(address string, port int, reporter Reporter) PortChecker {
+
+	log.Infof("Start checking address %s:%v", address, port)
+
+	go func() {
+		for true {
```

Comment:
Any reason to not use a `daemon.Daemon` for this loop?

Reply:
That's the next step. Just wanted it working first. Will do that before moving out of draft 👍